### DISCUSSION TITLE
Integration with WP Fluent Forms

### DISCUSSION THREAD

**Discussion by jdy98p** [pods-framework/pods#6598] [Ideas]
**Is your feature request related to a problem? Please describe.**
WP Fluent Forms has a free version and a paid version. It is *much* easier to work with than Gravity Forms and is rapidly adding functionality. I’d love to add Pods CPT fields to the list of options. 

**Describe the solution you'd like**
I’d like to be able to populate and map form fields in WP Fluent Forms similar to the way the Pods Plus Gravity Forms plugin works. 

**Describe alternatives you've considered**
Currently there are no alternatives I’ve found. 

**Additional context**

**Comment by jamierobe** [comment]
I would also like to see more form builder integrations. Fluent Forms has a very generous and feature rich free version. Thanks for the consideration, Jamie
